distbench/Makefile.am

```diff
@@ -1,7 +1,11 @@
 bin_PROGRAMS = bkjg margalit sshmidt
 
 # If you need special libraries, e.g. libm (math lib), declare them like so:
-# xxx_LIBS = -lm
+bkjg_LDFLAGS = -lm
+
+margalit_LDFLAGS = -lm
+
+sshmidt_LDFLAGS = -lm
 
 bkjg_SOURCES = main.c function.h bkjg.c
```

Comment:
The `function.h` file has been deleted. The "release ready" check fails, because it it still mentioned here.

Reply:
Oh, thanks, I see. I fixed it